### PR TITLE
Add logstash-monasca role that publish logs from Kafka to Elasticsearch.

### PR DESCRIPTION
**Pull request by tpolekhin**
Add logstash-monasca role that publish logs from Kafka to Elasticsearch.

### CODE REVIEW

File: roles/logstash-monasca/files/logstash.conf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+output {
+  elasticsearch {
+    codec => "json"
+    hosts => [ "elasticsearch-client-mantl.service.consul:9200" ]
+    index => "logs-%{+YYYY.MM.dd}"
+    template => "./logstash-template.json"
+    template_name => "logs"
+  }
+}
```
Comment (ryane) [inline roles/logstash-monasca/files/logstash.conf]: inputs and outputs should be configurable

**Comment by ryane** [comment]
as with #1783, I am wondering if this is appropriate for open source mantl. this looks similar to #1768 - if this stays in the mantl repo, can these become one configurable role? also, this would need a readme that describes what this role is, what you would use it for, how to install it, what it depends on, how to customize it, etc.

**Comment by tymofii-polekhin** [comment]
Updated

**Comment by siddharthist** [comment]
@tymofii-polekhin This still needs documentation - and discussion about the open/closed source issues -- any feedback @stevendborrelli?

**Pull request merged**